nifi-commons/nifi-flow-encryptor/pom.xml

```diff
@@ -34,5 +34,39 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.13.1</version>
```

Comment:
The version is managed through `jackson-bom` in the root Maven configuration, so this can be removed.
```suggestion
```

Reply:
Making the changes